test/unit/org/apache/cassandra/schema/MigrationCoordinatorTest.java

```diff
@@ -96,6 +100,8 @@ public class MigrationCoordinatorTest
             throw new AssertionError(e);
         }
 
+        CassandraRelevantProperties.SCHEMA_PULL_BACKOFF_INTERVAL_MS.setLong(0);
```

Comment:
So this is optimization not to use the default value. as it does not make sense in these tests?

Reply:
This test is very specific - it investigates different behaviours of `MigrationCoordinator` and sending subsequent requests in particular. We have it under control so we can set it to 0 to run it faster. 

In other tests, we want that backoff because resending a request is a part of normal operation and we don't want to flood the logs

Reply:
TBH, this is not a big deal - I can remove this property entirely - it just makes this https://github.com/apache/cassandra/pull/2972/files#diff-019068c4b04d4526506dae4c4a43dfde22aac4d1d74aa317cac755ae519ffe3aR347 wait shorter by one second